src/interceptors/ClientRequest/NodeClientRequest.ts

```diff
@@ -150,7 +149,7 @@ export class NodeClientRequest extends ClientRequest {
   end(...args: any): this {
     this.logger.info('end', args)
 
-    const requestId = uuidv4()
+    const requestId = crypto.randomUUID()
```

Comment:
Can you please import `crypto` from `node:crypto`? Afaik, it's not a universally supported global yet.

Reply:
Ok.